pycaret/internal/pycaret_experiment/time_series_experiment.py

```diff
@@ -139,6 +140,7 @@ def _get_setup_display(self, **kwargs) -> Styler:
                 ["Seasonal Period Tested", self.seasonal_period],
                 ["Seasonality Detected", self.seasonality_present],
                 ["Seasonality Used in Models", self.sp_to_use],
+                ["Seasonalities Used in Multiple Seasonality Models", self.all_sp_values],
```

Comment:
If you do not mind, can you rename this to the following:
```python
["Seasonal Period(s) Tested", self.seasonal_period],
["Seasonality Present", self.seasonality_present],
["Seasonalities Detected", self.all_sp_values],
["Primary Seasonality", self.sp_to_use],
```

Reply:
Sure!